.github/workflows/ci.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.8"]
-        nautobot-version: ["1.5.13"]
+        nautobot-version: ["v2.0.0-beta.1"]
```

Comment:
Please move to beta 2 now that it has been published. :)

Reply:
I was intending on doing a separate release for that but sure, I can get it updated for b2.

Reply:
Updated to use beta2 as requested.